frontend/public/components/utils/volume-kebab.tsx

```diff
@@ -0,0 +1,37 @@
+import * as _ from 'lodash-es';
+import * as React from 'react';
+import {
+  Kebab,
+  KebabOption,
+} from './index';
+import {
+  K8sKind,
+  K8sResourceKind,
+  K8sResourceKindReference,
+} from '../../module/k8s';
+import { connectToModel } from '../../kinds';
+
+export const VolumeKebab = connectToModel((props: VolumeKebabProps) => {
+  const {actions, kindObj, resource, isDisabled, volume} = props;
+  if (!kindObj) {
+    return null;
+  }
+  const options = _.reject(actions.map(b => b(kindObj, resource, volume)), 'hidden');
+  return <Kebab
+    options={options}
+    key={resource.metadata.uid}
+    isDisabled={isDisabled !== undefined ? isDisabled : _.get(resource.metadata, 'deletionTimestamp')}
+  />;
+});
+
+
+export type KebabVolumeAction = (kind: K8sKind, obj: K8sResourceKind, vol: any) => KebabOption;
```

Comment:
It's `VolumeKebab` above but `KebabVolumeAction` here. We should be consistent. I'd use `VolumeKebab` and `VolumeKebabAction`.